basyx/aas/adapter/json/aasJSONSchema.json

```diff
@@ -105,7 +105,7 @@
           "$ref": "#/definitions/AssetKind"
         },
         "globalAssetId": {
-          "$ref": "#/definitions/Reference"
+          "$ref": "#/definitions/Identifier"
```

Comment:
This change is not related to this PR, an already superseded by #70. Can you remove it?

Reply:
@dxvidnrt If this has been resolved, you can click "resolve conversation" here